esignet-core/src/main/java/io/mosip/esignet/core/dto/vci/ParsedAccessToken.java

```diff
@@ -8,5 +8,6 @@
 public class ParsedAccessToken {
 
     private Map<String, Object> claims;
+    private String atHash;
```

Comment:
where are we using this ?

Reply:
We need this for mosip IDA plugin support